CONTRIBUTING.md

```diff
@@ -28,6 +28,59 @@ Write some features. Run the tests with:
 yarn test
 ```
 
+## How to test locally
+
+The following details how to setup your local environment when you wish to create a new branch of a theme script package.  It uses work on a new release of `@shopify/themes-a11y` as an example.
+
+### Create a link to your local package to another project
+Navigate to the package and type [`yarn link`](https://yarnpkg.com/en/docs/cli/link).
+
+``` bash
+theme-scripts $ cd packages/themes-a11y
+theme-a11y $ yarn link
+
+# success Registered "@shopify/theme-a11y".
+# info You can now run `yarn link "@shopify/theme-a11y"`
+```
+
+Go to a local project you want to test out the script in.
+
+``` bash
+cd ../some-path/my-theme
+my-theme $ yarn link "@shopify/theme-a11y"
+
+# success Using linked package for "@shopify/theme-a11y".
+```
+
+**Note**: Do not run a new `yarn install`. This will reset your project to use the packages set in your `package.json`, which may replace your linked package.
```

Comment:
Fari question :D I'll test again because while I was writing this, I was also having an issue deploying to multiple environments in my project theme. So it may be unrelated. Will report back.

Reply:
It doesn't. Removing this :)